pkg/KubeArmorAnnotation/config/samples/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: myapp
+
```

Comment:
remove the empty line

Reply:
left over files, removed the whole samples directory